README.md

```diff
@@ -19,7 +19,15 @@ These packages can easily be installed by using one of the [AUR helpers](https:/
 
 1. Update `zividVersion` in `scripts/generate_all_pkgbuilds.sh`
 2. Run the tests. See [azure-pipelines.yml](azure-pipelines.yml).
+3. Make sure you have an AUR account with write access to the zivid repo and SSH public key.
+#### If you are on Archlinux
 3. Run `./scripts/update_aur.sh`.
+#### If you are on a different distro (e.g. Ubuntu)
+3. Build the deployment image: `docker build -t arch-deployment -f deployment/Dockerfile .`
+4. Start container while mounting your SSH keys and git config:
+    * `docker run --rm -v ~/.ssh:/home/deploymentuser/.ssh -v ~/.gitconfig:/home/deploymentuser/.gitconfig -it arch-deployment`
+6. From inside the container run `./zivid-arch-linux-pkgbuild-generator/scripts/update_aur.sh`
```

Comment:
Keep as much info as possible as code and keep readme simple. Free form text documentation tends to be or become wrong. I suggest keeping it as simple as this.

```
#### If you are on Archlinux
4. Run `./scripts/update_aur.sh`.
```
(note the change from 3 to 4)

Then I suggest changing `update_aur.sh` like this. (Untested, but it should be close):

```
# Run in docker if not on Arch
source /etc/os-release
if [ $ID != "arch" ]; then
    cd $(dirname "${BASH_SOURCE[0]}")/..
    docker build -t arch-deployment -f deployment/Dockerfile . || exit $?
    docker run --rm \
        -v ~/.ssh:/home/deploymentuser/.ssh \
        -v ~/.gitconfig:/home/deploymentuser/.gitconfig \
        -it arch-deployment \
        "$0" "$@"
    exit $?
fi
```

Reply:
For some reason I never noticed this feedback. Will get on it now, as it became relevant again.

Reply:
Check it out now. 
I just used it to deploy 2.3 to AUR from Ubuntu, so we know it works.